cirq-core/cirq/circuits/circuit_operation.py

```diff
@@ -94,6 +95,8 @@ class CircuitOperation(ops.Operation):
             will have its path prepended with the repetition id for each
             repetition. When False, this will not happen and the measurement
             key will be repeated.
+        repeat_until: A condition that will be tested after each iteration of
+            the circuit.
```

Comment:
This should clarify that the subcircuit repeats until this condition is true, and include a note that it is incompatible with repetitions.